tasks/update-component-sbom/tests/test-update-component-sbom-basic.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-update-component-sbom-basic
+spec:
+  description: |
+    Update a component-level SBOM where components contain only one purl.
```

Comment:
Reading this, I would expect to see some a test that there is only one purl. But it seems that's just what the update script does, right? And we don't test it here - because the script is not called, it's mocked. So I would suggest to say what the test does, e.g.:

Run the update-component-sbom task and verify that the update_component_sbom script is called as expected.